source/Mlos.Agent.Server/MlosAgentServer.cs

```diff
@@ -230,6 +294,13 @@ private class CliOptions
 
             [Option("optimizer-uri", Required = false, Default = null, HelpText = "A URI to connect to the MLOS Optimizer service over GRPC (e.g. 'http://localhost:50051').")]
             public Uri OptimizerUri { get; set; }
+
+            /// <remarks>
+            /// Just used to detect any extra arguments so we can throw a warning.
+            /// See Also: https://github.com/microsoft/MLOS/issues/112.
+            /// </remarks>
+            [CommandLine.Value(0)]
+            public IEnumerable<string> ExtraArgs { get; set; }

```

Comment:
This option unfortunately isn't hidden and leaves a weird `  value pos. 0 ` text at the end of the help output.